### PR TITLE
Skip Horovod 0.26 installation, add packaging to requirements.txt

### PR DESCRIPTION
**Pull request by arnavgarg1**

### CODE REVIEW

File: requirements_distributed.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ dask[dataframe]
 pyarrow==6.0.1 # https://github.com/ray-project/ray/issues/22310
 
 # requirements for horovod
-horovod[pytorch]>=0.24.0
+horovod[pytorch]>=0.24.0,<0.26.0
```
Comment (tgaddair) [inline requirements_distributed.txt:6]: You shouldn't need to pin horovod if you have the packaging requirement.

Comment (arnavgarg1) [reply requirements_distributed.txt:6]: I tried adding packaging just before the horovod install but it still caused the same error [add packaging to distributed](https://github.com/ludwig-ai/ludwig/pull/2642/commits/6c910c1e19aa2d7e7da5ad2e88ccfe61c5cede1a)

Comment (arnavgarg1) [reply requirements_distributed.txt:6]: Got it! Let me add that in

Comment (arnavgarg1) [reply requirements_distributed.txt]: Didn't actually know we could do `!=` in requirements.txt to make sure a certain version was never installed

**Comment by tgaddair** [comment]
Hopefully we should be able to get v0.26.1 out today:

https://github.com/horovod/horovod/pull/3741

**Review by tgaddair — APPROVED** [review]

**Pull request merged**